apps/research-service/src/site-polygons/dto/site-polygon-query.dto.ts

```diff
@@ -30,6 +30,16 @@ export class SitePolygonQueryDto extends IntersectionType(CursorPage) {
   @IsArray()
   projectId?: string[];
 
+  @ApiProperty({
+    name: "siteId[]",
+    isArray: true,
+    required: false,
+    description: "Filter results by site UUID(s). If specified, the includeTestProjects param is ignored"
```

Comment:
Please update the docs on `siteId`, `projectId` and `includeTestProjects` to indicate that they're mutually exclusive. Something like instead of the second sentence here, putting `Only one of siteId, projectId and includeTestProjects may be used in a single request`. And then I think throwing a `BadRequestException` in the builder if it sees that more than one of them is present would be a good addition. This is getting complicated enough that it deserves some guardrails.